src/mongooseim.app.src

```diff
@@ -14,6 +14,7 @@
                   cowboy,
                   cpool,
                   crypto,
+                  fast_scram,
```

Comment:
It is a stateless library, so I don't know if it is required to request otp to start it here. Is there some convention about the `applications` section maybe I'm missing? 🙂

Reply:
There is no need to start the app, on the other hand, the app needs to be included in the release. I think this is the clearest way.